internal/cli/delete.go

```diff
@@ -74,6 +106,23 @@ func RunDelete(kind, name string) error {
 			}
 			fmt.Printf("%s\n", response.Status())
 		}
+	case TemplateVersionKind:
+		if len(name) > 0 {
+			response, err := c.DeleteTemplateVersionWithResponse(context.Background(), *fleetName, name)
```

Comment:
I wish I had a real context  0:)
```suggestion
			response, err := c.DeleteTemplateVersionWithResponse(ctx, *fleetName, name)
```

Reply:
Fixed for all!